samples/ExampleFullWebApplication/ExampleFullWebApplication.vbproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="MSBuild.SDK.SystemWeb">
+﻿<Project Sdk="MSBuild.SDK.SystemWeb/4.0.88">
```

Comment:
This is set with global.json

Reply:
Thanks for catching that, I had run an experiment with MSBuildViewer to check something and didn't remove that inline SDK Version specification.